CIE-5515 - Release note for SELinux causing issues with an offline install (#4034)
SoftwareAG/c8y-docs: content/change-logs/edge/edge-2025.0.11-selinux.md

---
date: 2024-01-12

> title: The c8yedge tool is able to perform an offline installation with SELinux enabled

Reviewer: ```suggestion
title: The c8yedge tool can perform an offline installation with SELinux enabled
```

change_type:
  - value: change-VSkj2iV9m
    label: Fix
product_area: Edge
component:
  - value: component-g7hnfbu4J
    label: Edge
build_artifact:
  - value: tc-nJH2U7g3u
    label: edge
ticket: CIE-5515
version: 2025.0.11
---
When installing {{< product-c8y-iot >}} Edge with the *c8yedge* tool using an offline package, problems were sometimes observed when SELinux was enabled on the host system. This issue is dependent on the Linux distribution that is installed on the host system, but been observed on Rocky Linux 8 and Alma Linux 10.

> Firstly, *c8yedge* could attempt to download SELinux support packages from the internet, even if the customer had already installed them. Secondly, the K3s component used for hosting Edge could fail to start due to a permission error.

Author: @BeateRixen - The two of us discussed the formatting of the tool name (c8yedge) earlier, and concluded on italics. Do we want to change this, or should we introduce a change into the style guide?

> Firstly, *c8yedge* could attempt to download SELinux support packages from the internet, even if the customer had already installed them. Secondly, the K3s component used for hosting Edge could fail to start due to a permission error.

Reviewer: As far as I remember, we agreed to only highlight it once in a context?  I think you used bold formatting when introducing the term? Is a tool really to be treated as a command? Then we could change this but for me it is something different.

> Firstly, *c8yedge* could attempt to download SELinux support packages from the internet, even if the customer had already installed them. Secondly, the K3s component used for hosting Edge could fail to start due to a permission error.

Author: Ah yes, should be bold, not italics. How about c86eaa899396f5a48567f7575ea64da5fed2bfff ?

> Firstly, *c8yedge* could attempt to download SELinux support packages from the internet, even if the customer had already installed them. Secondly, the K3s component used for hosting Edge could fail to start due to a permission error.

Reviewer: This is consistent with what we have decided earlier, let´s leave it for now.
